### PR TITLE
📦️ Use pnpm 8.5.1 while generating lock files #1715

### PR DESCRIPTION
**Pull request by fernandolucchesi**

### CODE REVIEW

**Review by millianapia — APPROVED** [review]
lgtm

File: web/pageComponents/topicPages/PromoTileArray.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { BackgroundContainer, Card } from '@components'
 import { tokens } from '@equinor/eds-tokens'
 import { PortableTextBlock } from '@portabletext/types'
-import { CSSProperties, Fragment } from 'react'
+import { CSSProperties } from 'react'
```
Comment (SvSven) [inline web/pageComponents/topicPages/PromoTileArray.tsx:4]: Hmm, yes, pnpm version 🕵️‍♂️

**Review by SvSven — APPROVED** [review]
Good, to me looks. Hmmm.

File: web/pageComponents/topicPages/PromoTileArray.tsx (hunk 1 above)
Comment (fernandolucchesi) [reply web/pageComponents/topicPages/PromoTileArray.tsx:4]: hehehe fixed a few eslint warnings altogether

**Pull request merged**